### PR TITLE
Changelog bug links

### PR DESCRIPTION
**Pull request by saundefined**
Bug and feature request plain text numbers replaced by functions

### CODE REVIEW

File: ChangeLog-8.php [hunk 1]
```diff
@@ -231,7 +231,7 @@
 </ul></li>
 <li>Phpdbg:
 <ul>
-  <li>Reverted fix for bug #76813 (Access violation near NULL on source operand).</li>
+  <li>Reverted fix for bug <?php bugl(80600); ?> (Access violation near NULL on source operand).</li>
```
Comment (cmb69) [inline ChangeLog-8.php:234]: Wrong bug number.

**Review by cmb69 — COMMENTED** [review]
Nice, thank you! Besides the one issue, this looks good to me.

It might be worthwhile to add the replacement of "Implemented FR" to https://github.com/php/web-php/blob/master/bin/news2html

File: ChangeLog-8.php (hunk 1 above)
Comment (saundefined) [reply ChangeLog-8.php]: @cmb69, oops.. thank you! 👀 
Also added `GitHub PR` replacement

**Review by cmb69 — APPROVED** [review]
Thanks!

**Pull request merged**